playbook/app/pb_kits/playbook/pb_text_input/_text_input.scss

```diff
@@ -4,12 +4,12 @@
 
 
 [class^=pb_text_input_kit] {
+  margin-bottom: $space_sm;
```

Comment:
@JelenaD1 Can we have spacing powered by the spacing props in the kit instead of hardcoded in the sass? See how the card and popover kits handle this. They add/keep the default spacing, but if any spacing props are added those take precedence.

Secondly, this one has been flagged for alpha testing because we are moving the location where the spacing is added. This might now cause a problem but we need to confirm that by testing in Nitro. Request an alpha from the @jasperfurniss when you are ready to start that testing, or you can leave the spacing at the same level to skip alpha testing. Up to you. Just letting you know the next steps.

Reply:
Reason why I moved it is because story says spacing should be moved to the parent element allowing developer to override the existing spacing, which can be done by adding a global prop `margin_bottom: "lg"` to the kit.  @jasperfurniss and I talked about this on Friday and we need to discuss more on possible edge cases.

Reply:
@jasoncypret

Reply:
@JelenaD1 what is the current status here? Does @jasoncypret need to re-review? LGTM

Reply:
Remember we alpha tested this one together?  @thestephenmarshall